### PR TITLE
tidb-in-kubernetes: make procedures clearer for aws deployment

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 

This PR makes procedures clearer for aws deployment in `aws-eks.md`.

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1881

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

**Comment by anotherrachel** [comment]
@yikeke @DanielZhangQD PTAL, thanks~

File: dev/tidb-in-kubernetes/deploy/aws-eks.md [hunk 1]
```diff
@@ -249,7 +248,35 @@ The terraform scripts provide proper default settings for the TiDB cluster in EK
 
 For example, the default cluster uses `./default-cluster.yaml` as the overriding values file, and the ConfigMap rollout feature is enabled in this file.
 
-In EKS, some values are not customizable in `values.yaml`, such as the cluster version, replicas, node selectors, and taints. These variables are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. To customize these variables, you can edit the variables of each `./tidb-cluster` module in the `clusters.tf` file directly.
+In EKS, some values are not customizable in `values.yaml`, such as the cluster version, replicas, `NodeSelector` and `Tolerations`. `NodeSelector` and `Tolerations` are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. Cluster version and replicas can be modified in each `./tidb-cluster` module in the `clusters.tf` file directly.
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/deploy/aws-eks.md]: `./tidb-cluster` -> `tidb-cluster`

File: dev/tidb-in-kubernetes/deploy/aws-eks.md [hunk 2]
```diff
@@ -319,6 +346,10 @@ output "example-cluster_monitor-hostname" {
 }
 ```
 
+When you finish modification, you can execute `terraform init` and `terraform apply` to create a TiDB cluster.
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/deploy/aws-eks.md]: "create a TiDB cluster" -> "create the TiDB cluster"

File: dev/tidb-in-kubernetes/deploy/aws-eks.md [hunk 3]
```diff
@@ -319,6 +346,10 @@ output "example-cluster_monitor-hostname" {
...
+
+Once you remove `tidb-cluster` module, the corresponding TiDB cluster will be destroyed and EC2 resources will be released accordingly.
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/deploy/aws-eks.md]: "Once you remove" -> "If you remove"

File: dev/tidb-in-kubernetes/deploy/aws-eks.md [hunk 4]
```diff
@@ -249,7 +248,35 @@ The terraform scripts provide proper default settings for the TiDB cluster in EK
 
 For example, the default cluster uses `./default-cluster.yaml` as the overriding values file, and the ConfigMap rollout feature is enabled in this file.
 
-In EKS, some values are not customizable in `values.yaml`, such as the cluster version, replicas, node selectors, and taints. These variables are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. To customize these variables, you can edit the variables of each `./tidb-cluster` module in the `clusters.tf` file directly.
+In EKS, some values are not customizable in `values.yaml`, such as the cluster version, replicas, `NodeSelector` and `Tolerations`. `NodeSelector` and `Tolerations` are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. Cluster version and replicas can be modified in each `tidb-cluster` module in the `clusters.tf` file directly.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/deploy/aws-eks.md]:
```suggestion
In EKS, some configuration items are not customizable in `values.yaml`, such as the cluster version, replicas, `NodeSelector` and `Tolerations`. `NodeSelector` and `Tolerations` are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. Cluster version and replicas can be modified in each `tidb-cluster` module in the `clusters.tf` file directly.
```

**Comment by anotherrachel** [comment]
@yikeke PTAL

**Pull request merged**